app/controllers/matches.js

```diff
@@ -40,3 +41,13 @@ exports.getMatchHistory = (req, res, next) => {
     next(err);
   });
 };
+
+exports.sendMailOfUserHistory = (req, res, next) => {
+  const user = req.user;
+  if (!user.isAdmin && parseInt(req.params.user_id) !== user.id) {
+    next(errors.noAuthorizationError('User is not allowed'));
+  } else {
+    matchesMailer.sendUserHistory(user, req.params.user_id);
+    res.send('Email sent');
```

Comment:
Actually we should respond something like `Email will we delivered soon`, `Sending Email` or something like this. But It isn't really important right now.